src/functionalTest/resources/scenarios/RIA-2774-force-case-progression-notification.json

```diff
@@ -0,0 +1,97 @@
+              "parties": "legalRepresentative",
+              "dateDue": "{$TODAY+28}",
+              "dateSent": "{$TODAY}",
+              "tag": "requestCaseBuilding"
+            }
+          }
+        ],
+        "notificationsSent": [
+          {
+            "id": "2774_RESPONDENT_FORCE_CASE_PROGRESSION",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "2774_LEGAL_REPRESENTATIVE_REQUEST_CASE_BUILDING",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ],
+        "legalRepresentativeEmailAddress": "{$TEST_LAW_FIRM_A_USERNAME}"
+      }
+    },
+    "notifications": [
+      {
+        "reference": "2774_RESPONDENT_FORCE_CASE_PROGRESSION",
+        "recipient": "{$IA_RESPONDENT_REVIEW_DIRECTION_EMAIL}",
+        "subject": "Immigration and Asylum appeal: Case progressed without respondent evidence",
+        "body": [
+          "PA/12345/2019",
+          "CASE001",
+          "A1234567",
+          "Talha Awan"
+        ]
+      },
+      {
+        "reference": "2774_LEGAL_REPRESENTATIVE_REQUEST_CASE_BUILDING",
+        "recipient": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "subject": "Asylum appeal: build your case direction",
+        "body": [
+          "PA/12345/2019",
+          "CASE001",
+          "Talha Awan",
+          "You must now build your case by uploading your Appeal Skeleton Argument and evidence. You have 28 days from the date of this email to complete this.",
+          "You must write a full skeleton argument that references:",
+          "all the evidence you have (or plan) to rely on, including witness statements",
+          "the grounds and issues of the case",
+          "any new matters",
+          "any legal authorities you plan to rely on and why they are applicable to your case",
+          "Your argument must explain why you believe the respondent’s decision is wrong. You must provide all the information for the respondent to conduct a thorough review of their decision.",
+          "Next steps",
+          "Once you've uploaded your appeal argument and evidence, you should submit your case. The Tribunal case worker will review everything you've added.",
+          "If your case looks ready, the Tribunal case worker will send it to the respondent to review.",
```

Comment:
Notification body should only test the fields that are being inserted, rather than the whole body of the notification.

Reply:
I resued that from the existing functional test that covers the legal reps notification . That is the custom message sent to the legal rep. I think it makes to assert the message is the expected one